classes/blocks/class-timeline.php

```diff
@@ -59,6 +63,9 @@ public function __construct() {
 				],
 			]
 		);
+
+		add_action( 'enqueue_block_editor_assets', [ self::class, 'enqueue_editor_assets' ] );
```

Comment:
This function currently will load a fixed set of files, even if there's nothing to put in them. We should probably split it up so that we don't need to serve an empty file for this block https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/563/files#diff-7703839d1af73e6cde77fab17eeb77ed2f90c57a4559c3f45bf73795178e1fae

Reply:
I guess I need to adapt `entryPoint` function in the webpack config too

Reply:
Indeed. Would be nice if we could somehow write both in the same place :thinking:

Reply:
Though maybe not worth it as these things will not change that often.

Reply:
We could have a function per enqueue, so `enqueue_editor_scripts`, `enqueue_editor_styles`, `enqueue_frontend_scripts`, `enqueue_frontend_styles`.